src/benchmark/micro_benchmark_utils.hpp

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
 #include <stdint.h>
+#include <string>
 #include <vector>
 
 namespace hyrise {
 
 void micro_benchmark_clear_cache();
 void micro_benchmark_clear_disk_cache();
 std::vector<uint32_t> generate_random_indexes(uint32_t number);
+std::vector<uint32_t> generate_random_numbers(uint32_t size);
```

Comment:
Maybe we could name it more precisely to something that incorporates the fact that only positive numbers are generated?
```suggestion
std::vector<uint32_t> generate_random_positive_numbers(uint32_t size);
```